src/user/views/audit_views.py

```diff
@@ -450,3 +452,15 @@ def _send_email_notification_to_content_creator(self, flag, notification, verdic
             "flagged_and_removed_content.html",
             "ResearchHub Digest <digest@researchhub.com>",
         )
+
+    def _get_anonymous_remover(self):
+        user = User.objects.filter(email="community@researchhub.com")
+        if user.exists():
+            return user.first()
+
+        user = User.objects.filter(email="bank@researchhub.com")
```

Comment:
why do we have this as an anonymous remover?

Reply:
The data will return the editor who removed the notification. Since the action object contains that information, we can safely remove it from the notification object.

Reply:
I guess I'm not understanding the mechanism why we're doing user.first()

Reply:
This is a fail safe for dev environments since not all local environments has an "anonymous user"

Reply:
This is a fail safe for dev environments since not all local environments has an "anonymous user"